src/states/joining_peer.rs

```diff
@@ -43,12 +43,13 @@ pub struct JoiningPeer {
     network_service: NetworkService,
     routing_msg_filter: RoutingMessageFilter,
     msg_backlog: Vec<SignedRoutingMessage>,
```

Comment:
I would rename this one to `routing_msg_backlog` now that we have `direct_msg_backlog` too, for explicitness.